src/gasx.f90

```diff
@@ -596,9 +596,9 @@ SUBROUTINE o2sato(T, S, N, o2sat_molm3)
 
 ! INPUT variables
   !> surface temperature [C]
-  REAL(kind=r4), INTENT(in), DIMENSION(N) :: T
+  REAL(kind=r8), INTENT(in), DIMENSION(N) :: T
   !> surface salinity [psu]
-  REAL(kind=r4), INTENT(in), DIMENSION(N) :: S
+  REAL(kind=r8), INTENT(in), DIMENSION(N) :: S
```

Comment:
This and the one above are the same as https://github.com/jamesorr/mocsy/pull/3/files#r61793559